### PR TITLE
refactor(console): add sie tab constants

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

Add sie tab constants to avoid potential handwriting mistakes.

## Testing

Test locally.

### CODE REVIEW

File: packages/console/src/consts/page-tabs.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export const signInExperiencePage = {
+  brandingTab: 'branding',
+  signUpAndSignInTab: 'sign-up-and-sign-in',
+  othersTab: 'others',
+};
```
Comment (simeng-li) [inline packages/console/src/consts/page-tabs.ts:5]: nit: enum?

Comment (simeng-li) [reply packages/console/src/consts/page-tabs.ts:5]: or object.freeze so it may has type guard

Comment (xiaoyijun) [reply packages/console/src/consts/page-tabs.ts]: updated with enum

**Review by simeng-li — APPROVED** [review]
LGTM

**Pull request merged**